libbuild2-autoconf/libbuild2/autoconf/checks/HAVE__BOOL.h

```diff
@@ -0,0 +1,14 @@
+// HAVE__BOOL
+
+#undef HAVE__BOOL
+
+/* Check for the C99 _Bool type.
+ *
+ * @@ TMP Although this check does look correct, HAVE_STDBOOL_H is defined to
+ *    1 unconditionally but <stdbool.h> defines bool to _Bool and thus
+ *    logically depends on _Bool being available. Thus HAVE_STDBOOL_H should
+ *    probably depend on this check but that might cause breakage.
```

Comment:
Hm, I don't think it's our job to make sure combinations of checks are not contradictory. If some platform doesn't have `_Bool` but has `<stdbool.h>`, I don't think we need to detect that.